### PR TITLE
[ci-skip] Translate to Chinese

### PR DESCRIPTION
**Pull request by haha252**
The wiki I will upload soon.

### CODE REVIEW

**Comment by haha252** [comment]
The Wiki
[安装方式(Installation du plugins).md](https://github.com/Euphillya/Skyllia/files/14034824/Installation.du.plugins.md)
[运作方式(Fonctionnement du plugin).md](https://github.com/Euphillya/Skyllia/files/14034829/Fonctionnement.du.plugin.md)
[权限管理(Permissions Standard du Plugin Skyllia).md](https://github.com/Euphillya/Skyllia/files/14034838/Permissions.Standard.du.Plugin.Skyllia.md)

**Comment by Euphillya** [comment]
Create a wiki folder at the plugin source, which will be used to store future languages as follows: 
wiki/lang/all_files

And we'll put all the wikis in the readme.md with a clickable link.

**Comment by haha252** [comment]
Like this?

**Comment by Euphillya** [comment]
Yes, ans résolve conflict with readme :)

**Comment by haha252** [comment]
Emmmm, I edited readme.md 
add a clickable button to Chinese readme file(readme_zh.md)

**Comment by Euphillya** [comment]
Configurations are no longer in the readme

**Comment by haha252** [comment]
finished

**Comment by haha252** [comment]
How About this one

File: readme_zh.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+		not-enough-args = "创建个传送点，你也得说一下你这个传送点叫啥呀？: /skyfolia <set/del>warp <warp_name>"
+
+		[island.warp.teleport]
+			not-exist = "你这个传送点好像不存在呀？"
+			success = "传送成功~"
+		[island.warp.delete]
+			success = "传送点删除成功~"
+			can-not-delete-home = "不能删除home这个传送点哦"
+
+```
```
Comment (Euphillya) [inline readme_zh.md:206]: Create the wiki page for this.

**Review by Euphillya — CHANGES_REQUESTED** [review]
Change that so I can start importing.Cliquez pour utiliser cette solution

**Comment by haha252** [comment]
awa

File: readme_zh.md (hunk 1 above)
Comment (haha252) [reply readme_zh.md:223]: ok for a moment

**Comment by haha252** [comment]
ok finished

**Comment by Euphillya** [comment]
It's not up to date with the new configs and API, but it'll be good enough!

**Pull request merged**